staking/types/validator.go

```diff
@@ -85,25 +125,27 @@ type ValidatorStats struct {
 // Validator - data fields for a validator
 type Validator struct {
 	// ECDSA address of the validator
-	Address common.Address
+	Address common.Address `json:"address"`
 	// The BLS public key of the validator for consensus
-	SlotPubKeys []shard.BlsPublicKey
-	// The number of the last epoch this validator is selected in committee (0 means never selected)
-	LastEpochInCommittee *big.Int
+	SlotPubKeys []shard.BlsPublicKey `json:"bls-public-keys"`
+	// The number of the last epoch this validator is
+	// selected in committee (0 means never selected)
+	LastEpochInCommittee *big.Int `json:"last-epoch-in-committee"`
 	// validator's self declared minimum self delegation
-	MinSelfDelegation *big.Int
+	MinSelfDelegation *big.Int `json:"min-self-delegation"`
 	// maximum total delegation allowed
-	MaxTotalDelegation *big.Int
-	// Is the validator active in participating committee selection process or not
-	Active bool
+	MaxTotalDelegation *big.Int `json:"max-total-delegation"`
+	// Is the validator active in participating
+	// committee selection process or not
+	Active bool `json:"eligible-for-epos"`
```

Comment:
let's use the original wording to be consistent.

Reply:
but the active field was making people confused and it doesn't capture the semantic implication that active is with respect to being able to be in epos auction